MSID_limit/compare_database_and_update.py

```diff
@@ -401,20 +403,20 @@ def test_and_save():
 #--- update the main mta limit database
 #
     if len(data) < 1:
-        cmd = 'rm '  + temp_opfile
+        cmd = f'rm {temp_opfile}'
```

Comment:
Not a huge deal, but all these `os.system` commands can be replaced by Python built-in functions in https://docs.python.org/3.8/library/shutil.html and https://docs.python.org/3.8/library/os.html. There are `move()` and `copy2()` in `shutil` and `remove` in `os`.

The main problem here is that there is no real error handling in the current `os.system(cmd)` call. If that fails then a non-zero value is returned but that is being ignored in this code. At the least all the `os.system(cmd)` calls should be replaced by `assert os.system(cmd) == 0` so an exception is raised for a problem.